clients/ts/signalr-protocol-msgpack/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aspnet/signalr-protocol-msgpack",
-  "version": "1.1.0-preview1-t000",
+  "version": "1.1.0-preview2-t000",
```

Comment:
This was missed in the branding update @natemcmaster did (our bad though, for not catching it in review). Maybe we should like... fail the build on the CI if this file changes. We need a way to make sure changes get checked in.